src/Public/OAComponents.ps1

```diff
@@ -63,10 +67,13 @@ function Add-PodeOAComponentResponse {
 
         [Parameter(ParameterSetName = 'Schema')]
         [System.Collections.Specialized.OrderedDictionary ]
-        $Links
+        $Links,
+
+        [string]
+        $SpecTag = 'default'
```

Comment:
You've had some fun naming this parameter 😂, would `$DefinitionName` with an alias for just `$DefName` work better?

"Tag" might get confusing with OATags, which I saw you had as one of the names previously. But also that 1 definition could have multiple "tags", rather than just a unique name like "default" 🤔

Reply:
I'll change to $defname.

If you have time please try the modified version Petstore-openApiMultiTag.ps1 
this file produce both 3.0 and 3.1 openAPI documents
Or if you want have fun try the OpenApiTuttiFrutti.ps1. The is everything there. it's an orgy of definitions with little sense but include any function I created for OpenAPI

Reply:
Name is not right maybe someting like DefinitionTag is more fit